From 3ba29cde19a9e66d3ad1a74d481356168e27ddab Mon Sep 17 00:00:00 2001
From: Cong Wang <xiyou.wangcong@gmail.com>
Date: Tue, 17 Apr 2012 21:25:11 +0800
Subject: [PATCH] network: move the kill-dhclient.sh hook later

kdump module uses a pre-pivot hook too, it needs networking, so
do not kill dhclient so early. kdump hook needs to find a place
where all needed fs are mounted meanwhile networking works too.

Cc: Harald Hoyer <harald@redhat.com>
Signed-off-by: Cong Wang <xiyou.wangcong@gmail.com>
---
 modules.d/40network/module-setup.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/module-setup.sh b/modules.d/40network/module-setup.sh
index 97fc5cb..49e440e 100755
--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -85,7 +85,7 @@ install() {
     inst_hook cmdline 97 "$moddir/parse-bridge.sh"
     inst_hook cmdline 98 "$moddir/parse-ip-opts.sh"
     inst_hook cmdline 99 "$moddir/parse-ifname.sh"
-    inst_hook pre-pivot 10 "$moddir/kill-dhclient.sh"
+    inst_hook pre-pivot 98 "$moddir/kill-dhclient.sh"
 
     _arch=$(uname -m)
 
